docs/custom-domain/README.md

```diff
@@ -1,105 +1,128 @@
+This article describes the steps to set up the endpoints of an Azure container registry using your own custom domain name, such as **contoso.com**.
 
-The following steps describe how you can achieve this.
+By default, every Azure container registry is accessed externally using its login server name. If you have a registry called `myregistry`, you access it using its default hostname, `myregistry.azurecr.io` (in the Azure Cloud). As a customer belonging to an organization, you may prefer to access your registry using a custom domain name that is associated with your organization, for example, `myregistry.contoso.com`.
+
+**Important - Using a custom domain name in Azure Container Registry is currently a private preview feature**. It can only be enabled by preparing certificates, a Premium container registry, and an Azure key vault as described in this article, and then opening an Azure support request to complete the configuration. 
+
+## Limitations
+
+* This capability is provided without a service level agreement, and isn't currently recommended for production workloads.
+* Azure Container Registry custom domains can't currently be used where key vault access is restricted. 
 
 ## Prerequisites
-- [Azure CLI](https://docs.microsoft.com/cli/azure/?view=azure-cli-latest): version 2.4.0 or higher
-  - Consider using [Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/overview)
-- A _premium_ Azure Container Registry. See [here](https://docs.microsoft.com/azure/container-registry/container-registry-get-started-azure-cli) for instructions on how to create one.
+- [Azure CLI](https://docs.microsoft.com/cli/azure/): version 2.4.0 or higher, or use [Azure Cloud Shell](https://docs.microsoft.com/azure/cloud-shell/overview)
+- An Azure container registry in the Premium service tier. See [this article](https://docs.microsoft.com/azure/container-registry/container-registry-get-started-azure-cli) for instructions to create one. If you have an existing registry, you can [upgrade](https://docs.microsoft.com/azure/container-registry/container-registry-skus#changing-tiers).
 - Your custom domain names. The following two are required:
   - Custom registry domain to access the registry REST API. Example for the `contoso.com` domain: `container-registry.contoso.com` 
-  - Custom data domain to access the registry content. Again, example for `contoso.com`: `eastus-registry-data.contoso.com`
-    - Note that the custom data domain is region specific. For geo-replicated registries, each region should have its own custom data endpoint.
+  - Custom data domain to access the registry content. Example for `contoso.com`: `eastus-registry-data.contoso.com`
```

Comment:
can we follow the same recommendation we've made for ACR:

- REST endpoint: registry.domain (registry.contoso.com)
- Data endpoint: [region].data.registry.contoso.com (eastus.data.registry.contoso.com, westus.data.registry.contoso.com)

Reply:
Good suggestion, updated.